index.html

```diff
@@ -1986,76 +1986,103 @@ <h1> Material beyond this point has not been assigned.  The descriptions below a
 </p>
 
 <p>
 <b>IMPORTANT (seriously):</b> The given robot description files should <b>NOT</b> be modified.  Code that requires modified robot description files will fail tests used for grading.  You are welcomed and encouraged to create new robot description files for additional testing.
 </p>
 
 <p>
-The selection of file with a robot description can occur directly in the URL for  "home.html".  As a default, the "home.html" in the KinEval stencil assumes the "mr2" robot description in "robots/robot_mr2.js".  Another robot description file can be selected directly in the URL by adding a robot parameter.  This parameter is segmented by a question mark and sets the robot file pointer to a given file local location, relative to "home.html".  For example, a URL with "home.html?robot=robots/robot_urdf_example.js" will use the URDF example description.
+The selection of different robot descriptions can occur directly in the URL for  "home.html".  As a default, the "home.html" in the KinEval stencil assumes the "mr2" robot description in "robots/robot_mr2.js".  Another robot description file can be selected directly in the URL by adding a robot parameter.  This parameter is segmented by a question mark and sets the robot file pointer to a given file local location, relative to "home.html".  For example, a URL with "home.html?robot=robots/robot_urdf_example.js" will use the URDF example description. Note that to see the selected robot model in your visualization, you will need to turn off Just Starting Mode and have your FK methods implemented; see the Invoking Forward Kinematics section for more details.
 </p>
 
+<h3> Initialization of Kinematic Hierarchy </h3>
 <p>
-In addition to the given initialization, you should extend the robot object to complete the kinematic hierarchy to specify the parent and children of each link.  This modification should be made in the kineval.initRobotJoints() function in "kineval/kineval_robot_init.js".  The children array of a link should always be defined, which would result in an empty array for leaf nodes in the kinematic tree.  For the KinEval user controls to work properly, the children array should be named as the ".children" property of the link.
+In addition to the various existing initialization functions, you should extend the robot object to complete the kinematic hierarchy to specify the parent and children joints for each link.  This modification should be made in the kineval.initRobotJoints() function in "kineval/kineval_robot_init_joints.js".  The children array of a link should be defined for all links except the leaves of the kinematic tree, in which case the ".children" property should be left undefined.  For the KinEval user controls to work properly, the children array should be named the ".children" property of the link.
```

Comment:
If you don't leave the .children undefined, they keyboard controls let you select joints that don't exist, which causes some weird issues with the visualization. I changed this to state that they should leave .children undefined instead of making it an empty array for leaf nodes. However, we could also fix this in the keyboard selection code if that's preferable.